tutorial/1_simple_end_to_end/RUNME.ipynb

```diff
@@ -21,15 +22,15 @@
      "output_type": "stream",
      "text": [
       "╔══════════════════════════════════════════════╦═════════╦══╦═══════════╦═══╦══════════════════════════════╦══════════╦══════════════════════════╦══════════╗\n",
-      "║ single partition (Part 1) cost (throughput): ║ 74515.6 ║  ║ slowdown: ║ 1 ║ partition optimisation time: ║ 0.06 sec ║ total optimisation time: ║ 0.06 sec ║\n",
+      "║ single partition (Part 1) cost (throughput): ║ 74543.4 ║  ║ slowdown: ║ 1 ║ partition optimisation time: ║ 0.08 sec ║ total optimisation time: ║ 0.08 sec ║\n",
```

Comment:
Maybe we don't need to track notebook outputs. It's currently a bit difficult to find out what actually is being changed. Other than that, looks good to me. Thanks！